index.html

```diff
@@ -48,6 +49,173 @@ <h4 class="social-title">LET'S CONNECT</h4>
+                
+                    <ul class="tech2">
+                        <li class="language1">html</li>
+                        <li class="language2"> css</li>
+                        <li class="language3">javascript</li>
+                    </ul>
+                        <button type="button" class="see-proj2">See Project</button>
+                </div> 
+            </li>
+            <!-- Third Card -->
+            <li class="card-container3">
+                <img src="./images/Snapshoot3.png" class="card-container-img3" alt="Snapshoot3">
+
+                <h2 class="title-project3">Tonic</h2>
+
+                <div class="card-content3">
+                    <ul>
+                        <li class="type3">
+                            <p class="t1">CANOPY</p>
+                            <p class="t2"><img src="./images/Circle.png"/> Back End Dev</p>
+                            <p class="t3"> <img src="./images/Circle.png"/> 2015</p>
+                        </li>  
+                    </ul> 
+                    
+                    <p class="text3">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    
+                    <ul class="tech3">
+                        <li class="language1">html</li>
+                        <li class="language2"> css</li>
+                        <li class="language3">javascript</li>
+                    </ul>
+
+                    <button type="button" class="see-proj3">See Project</button>
+                </div>
+            </li>   
+            <!-- Fourth Card -->
+            <li class="card-container4">
+                <img src="./images/Snapshoot4.png" class="card-container-img4" alt="Snapshoot4">
+
+                <h2 class="title-project4">Multi-Post <br> Stories</h2>
+
+                <div class="card-content4">
+                    <ul>
+                        <li class="type4">
+                            <p class="t1">CANOPY</p>
+                            <p class="t2"><img src="./images/Circle.png"/> Back End Dev</p>
+                            <p class="t3"> <img src="./images/Circle.png"/> 2015</p>
+                        </li>  
+                    </ul> 
+
+                    <p class="text4">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+
+                    <ul class="tech4">
+                        <li class="language1">html</li>
+                        <li class="language2"> css</li>
+                        <li class="language3">javascript</li>
+                    </ul>
+
+                    <button type="button" class="see-proj4">See Project</button>
+                </div>
+            </li>
+        </ul>
+    </section>
+                                    <!-- Section 2 : About Myself -->
+    <section class="about-myself">
+        <div class="about-container">
+            <h3 class="about-myself-title">About Myself</h3>
```

Comment:
- [ ] Kindly **as suggested by the previous reviewer, 👆 👆** I think it would be better to follow the  Figma design's color on the background, This will make your design more elegant ✨😉✨😉 

You may Kindly try to give the same background and this will make your design cleaner and clear 🙏 

![image](https://user-images.githubusercontent.com/102757126/219867228-32b5395f-1221-4ab7-bda9-dd2191cf300e.png)


- [ ] **[Optional]** I think it would be better to prevent scrolling horizontally in your CSS for the best User experience👍 👍 

example:
```
body {
  overflow-x: hidden;
  width: 100%;
}
```
_you may kindly follow this[ link ](https://www.w3schools.com/howto/howto_css_hide_scrollbars.asp)for more info on hidden overflow_